### DISCUSSION TITLE
端午節快樂🐉🫔🎉

### DISCUSSION THREAD

**Discussion by tshsu04224** [1101-GS4719-C/Python-Community-GS4719-C#2018] [General]
這次端午節連假想必很多人還沈浸在期末的水深火熱當中，不知道大家今年有沒有立蛋🥚或者吃個粽子呢，希望大家期末都順利，趕快擺脫期末地獄，開開心心放暑假🥳 是說不知道為什麼現在好像對節日有越來越無感的趨勢，是因為常常連假都藉著大考所以只能讀書嗎？有點好奇是大家都這樣還是只有我這樣🤔

**Comment by 155Julia** [comment]
今年端午節，雖然我沒有立蛋，但享受了一些美味的粽子。幸運的是，我這次期末主要是專題報告，壓力相對較小。這讓我有更多時間放鬆和享受節日。未來，我希望能更好地平衡學習和節日慶祝，讓每一個節日都能成為放鬆和重溫傳統的機會，而不僅僅是被學業壓力所占據。希望大家也能找到自己的節奏，在忙碌的學習生活中找到快樂的時光。

**Comment by YeeDaniel** [comment]
這完全我，完全不理解為甚麼連假還需要讀書... 明明就可以排在連假前考，真的是不能理解。再加上這學期的很多考試都擠在端午節後才考，對我來說真的壓力超大。幸好有一些專案報告的時間有往前排，有相對減輕我一點負擔

**Comment by yubo0oo** [comment]
端午連假要念一堆書，還要做很多報告，真的相當累人

**Comment by poiu6fi** [comment]
今年端午節要交兩份報告== 根本沒時間看到底有甚麼活動，整天都窩在電腦前做作業，只覺得又是一個坐牢的期末周
